tests/phpunit/tests/webfonts-api/mocks/class-my-custom-webfonts-provider-mock.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require_once ABSPATH . WPINC . '/webfonts-api/providers/class-wp-webfonts-provider.php';
+
+class My_Custom_Webfonts_Provider_Mock extends WP_Webfonts_Provider {
+	protected $id = 'my-custom-provider';
+
+	protected $link_attributes = array(
+		'href'        => 'https://fonts.my-custom-api.com',
+		'crossorigin' => true,
+	);
```

Comment:
Where do we use this property?

Reply:
There's an update in https://github.com/WordPress/wordpress-develop/pull/1736/commits/7939995d89f495560d29945b576292168084539f to correct the property name to match the providers.

This property is used to create the preconnect link:

```html
<link rel="preconnect" href="https://fonts.my-custom-api.com" crossorigin>
```

A test _used to_ exist for this, but was removed in a refactoring. Will add that test back in but connect it instead to the filter (i.e. `'wp_resource_hints'`) that triggers rendering it.